core-services-v2/prow/03_deployment/namespaces.yaml

```diff
@@ -0,0 +1,51 @@
+kind: List
```

Comment:
Applyconfig goes over files in lexicographical order so we should make sure this one goes first.